README.md

```diff
@@ -118,6 +118,20 @@ Have need for a custom or domain specific integer sequence? You simply need to m
     }
     ```
 
+5. Finally, use your `myCustomIntRange` to generate integers
+    ```javascript
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[list-item-content-indent] Don’t use mixed indentation for children, remove 1 space](https://app.codacy.com/manual/chalu/int-range/pullRequest?prid=5138485)